assets/js/common/ModifiedCheckPill/ModifiedCheckPill.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import Pill from '@common/Pill';
+
+function ModifiedCheckPill({ className, customized = false }) {
+  if (!customized) return null;
+  return (
+    <Pill
+      className={classNames(
+        'bg-white text-jungle-green-500 border border-jungle-green-500',
+        className
+      )}
+      size="xs"
+    >
+      MODIFIED
```

Comment:
We could have a label prop with defaul value MODIFIED, so the component is a bit more reusable, what do you think ?

Reply:
Done.